docs/configuration.md

```diff
@@ -19,7 +19,12 @@
 | MetricScrapingPeriodSeconds  | int       | > 0                                                                 | 15                  | Period of Prometheus metrics scrapping                                               |
 | GRPCConnectionTimeoutSeconds | int       | > 0                                                                 | 60                  | Timeout for establishing a gRPC connection                                           |
 | GRPCFunctionTimeoutSeconds   | int       | > 0                                                                 | 90                  | Maximum time given to function to execute[^4]                                        |
-| DAGMode             | bool      | true/false                                                          | false               | Sequential invocation of all functions one after another                                                    |
+| DAGMode                      | bool      | true/false                                                          | false               | Parallel invocation of DAG with each function acting as entry function                                                                                                   |
+| DAGTracePath                 | string    | string                                                              | data/traces/sampled_150/20               | Folder with Azure trace dimensions used for DAG Invocation.                                                                                                |
```

Comment:
Please add a reference to the trace. It can be located with the rest of the docs (see other comment).